EvidenceApi/StaffSelectedDocumentTypes.json

```diff
@@ -238,6 +268,171 @@
         "id": "private-pension-statement",
         "title": "Private pension statement",
         "description": "From within the last year"
```

Comment:
In the spreadsheet, occupation/private and state pensions don't seem to have descriptions, so they could be removed from the file.